api/routes/users.js

```diff
@@ -1,9 +1,42 @@
 var express = require('express');
+var Users = require('../../db/models/Users')
 var router = express.Router();
 
-/* GET users listing. */
-router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+const Users = require("../../db/models/Users");
+
+const Address = require("../../db/models/Address");
+
+const Organization = require("../../db/models/Organization");
+
+router.get("/", async (req, res, next) => {
+  const users = await Users.find();
+  return res.send(users);
 });
 
+router.post("/", async (req, res, next) => {
+  let addressBody = req.body.address;
+  const address = new Address({
```

Comment:
can we wrap somehow in some async await ? basically what if one of the three things fails? we want the user to be able to get some helpful feedback. 

this only works if all of these things work.

Reply:
addressed